internal/wasm/module.go

```diff
@@ -901,11 +917,28 @@ const (
 	ValueTypeI64 = api.ValueTypeI64
 	ValueTypeF32 = api.ValueTypeF32
 	ValueTypeF64 = api.ValueTypeF64
+	// TODO: RefTypes are not exposed in the api pkg yet.
+	ValueTypeFuncref   ValueType = 0x70
+	ValueTypeExternref ValueType = 0x6f
 )
 
 // ValueTypeName is an alias of api.ValueTypeName defined to simplify imports.
 func ValueTypeName(t ValueType) string {
-	return api.ValueTypeName(t)
```

Comment:
special-case the switch instead? ex. case ValueTypeFuncref: "funcref" default api.ValueTypeName(t)